<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BaseGroundStation</class>
 <widget class="QMainWindow" name="BaseGroundStation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AHNS Ground Station</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="dispNumber">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Display Number</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeBtn">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>80</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dispText">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>50</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Display Text</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="numDisp">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>120</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Filled</enum>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>351</width>
      <height>451</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QPushButton" name="confirmCB">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>340</y>
        <width>80</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>confirm</string>
      </property>
     </widget>
     <widget class="QLabel" name="txtDisp">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>270</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>This is a test</string>
      </property>
     </widget>
     <widget class="QLabel" name="decFloat">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>340</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>dec</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>310</y>
        <width>101</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>CheckBox</string>
      </property>
     </widget>
     <widget class="QPushButton" name="sendUDPmsg">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>120</y>
        <width>111</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>sendUDPmsg</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="inDisp">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>231</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextBrowser" name="outDisp">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>70</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="timedJoystick">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>170</y>
        <width>111</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>timedJoystick</string>
      </property>
     </widget>
     <widget class="QLabel" name="jsDisp">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>170</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Joystick Value</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QTextBrowser" name="statusLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>261</width>
        <height>71</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="UDPop">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>210</y>
        <width>92</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>UDPop</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="orgDatagram">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>261</width>
        <height>71</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="clearUDP">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>240</y>
        <width>92</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear UDP</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="textEnterUDP">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>261</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>170</y>
      <width>111</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>STOPWATCH</string>
    </property>
    <widget class="QLCDNumber" name="timerDisp">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Filled</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="timerStart">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>92</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="timerPause">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>92</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Pause</string>
     </property>
    </widget>
    <widget class="QPushButton" name="timerReset">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>92</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
